client/task-list/index.js

```diff
@@ -33,6 +33,29 @@ export class TaskDashboard extends Component {
 		document.body.classList.add( 'woocommerce-task-dashboard__body' );
 	}
 
+	trackStartedTask = ( taskName ) => {
+		const { trackedStartedTasks, updateOptions } = this.props;
+		const taskStarted = ( trackedStartedTasks || [] ).filter(
+			( task ) => task === taskName
+		);
+		if ( taskStarted.length > 1 ) {
```

Comment:
No particular reason, given the `trackedCompletedTasks` was also an array I followed suite with that. Originally I thought I would only ever need one item in the list, but that didn't work given the update didn't happen on the same page that had the show tooltip logic.

Suggestions otherwise? Maybe a key:number object, numbers being the amount a task was visited. The fact that I only needed to know the second time I didn't think it necessary to track after that.

Reply:
I'm partial to a key:number object, but I don't feel too strongly about it. One potential benefit of the object is that we might be able to extract value from the count .. perhaps send the number of times a step was viewed in the tracks event? That might tell us if particular tasks are confusing or difficult.

Reply:
Yeah not a bad idea, in that case I would have to change the count logic to update until it is finished. Instead of not more then 2.

Reply:
I'm just spitballing here - no need to add extra work unnecessarily.